### PR TITLE
Hindi locale if Rupee currency

### PR DESCRIPTION
**Pull request by raunak-smpx**

### CODE REVIEW

File: src/Str/Str.js [hunk 1]
```diff
@@ -156,7 +156,8 @@ function trimToNext(str, pos, char = ' ') {
 
 const numberLocaleCache = {};
 function _getNumberLocale(options) {
-	const {locale = 'en', currency = undefined, decimals = 0} = options;
+	const {currency = undefined, decimals = 0} = options;
+	const locale = (currency === 'INR' ? 'HIN' : options.locale) || 'en';
```
Comment (hit12369) [inline src/Str/Str.js]: `HIN` locale should be `hi-IN` to go by the convention of `language-country`.

Comment (hit12369) [reply src/Str/Str.js]: Also add a comment on why this is needed (due to space issue).

Comment (raunak-smpx) [reply src/Str/Str.js]: Done

File: src/Str/Str.js [hunk 2]
```diff
@@ -157,7 +157,8 @@ function trimToNext(str, pos, char = ' ') {
 const numberLocaleCache = {};
 function _getNumberLocale(options) {
 	const {currency = undefined, decimals = 0} = options;
-	const locale = (currency === 'INR' ? 'HIN' : options.locale) || 'en';
+	// If currency is INR, there should not be space between currency symbol and number
+	const locale = (currency === 'INR' ? 'HI-IN' : options.locale) || 'EN';
```
Comment (hit12369) [inline src/Str/Str.js]: Language should be lowercased. `en` and `hi`, while region should be uppercase `IN`.

**Pull request merged**